editor/editor_plugin_settings.cpp

```diff
@@ -179,12 +150,39 @@ void EditorPluginSettings::_cell_button_pressed(Object *p_item, int p_column, in
 	if (p_id == BUTTON_PLUGIN_EDIT) {
 		if (p_column == 4) {
 			String dir = item->get_metadata(0);
-			plugin_config_dialog->config("res://addons/" + dir + "/plugin.cfg");
+			plugin_config_dialog->config(dir);
 			plugin_config_dialog->popup_centered();
 		}
 	}
 }
 
+Vector<String> EditorPluginSettings::_get_plugins(const String &p_dir) {
+	DirAccessRef da = DirAccess::create(DirAccess::ACCESS_RESOURCES);
+	Error err = da->change_dir(p_dir);
+	if (err != OK) {
+		return Vector<String>();
+	}
+
+	Vector<String> plugins;
+	da->list_dir_begin();
+	for (String path = da->get_next(); path != String(); path = da->get_next()) {
+		if (path[0] == '.' || !da->current_is_dir()) {
```

Comment:
You can avoid this check by enabling `skip_navigational` in your `list_dir_begin`